apps/scan/backend/src/scanners/custom/state_machine.ts

```diff
@@ -181,45 +181,62 @@ async function closeCustomClient({ client }: Context) {
 function scannerStatusToEvent(
   scannerStatus: ScannerStatus
 ): ScannerStatusEvent {
-  const frontHasPaper =
+  const allFrontSensorsDetectPaper =
     scannerStatus.sensorInputLeftLeft === SensorStatus.PaperPresent &&
     scannerStatus.sensorInputCenterLeft === SensorStatus.PaperPresent &&
     scannerStatus.sensorInputCenterRight === SensorStatus.PaperPresent &&
     scannerStatus.sensorInputRightRight === SensorStatus.PaperPresent;
-  const backHasPaper =
-    scannerStatus.sensorOutputLeftLeft === SensorStatus.PaperPresent &&
-    scannerStatus.sensorOutputCenterLeft === SensorStatus.PaperPresent &&
-    scannerStatus.sensorOutputCenterRight === SensorStatus.PaperPresent &&
+  const someFrontSensorsDetectPaper =
+    scannerStatus.sensorInputLeftLeft === SensorStatus.PaperPresent ||
+    scannerStatus.sensorInputCenterLeft === SensorStatus.PaperPresent ||
+    scannerStatus.sensorInputCenterRight === SensorStatus.PaperPresent ||
+    scannerStatus.sensorInputRightRight === SensorStatus.PaperPresent;
+
+  const someBackSensorsUnderRollersDetectPaper =
+    scannerStatus.sensorOutputCenterLeft === SensorStatus.PaperPresent ||
+    scannerStatus.sensorOutputCenterRight === SensorStatus.PaperPresent;
+  const someBackSensorsPastRollersDetectPaper =
+    scannerStatus.sensorOutputLeftLeft === SensorStatus.PaperPresent ||
     scannerStatus.sensorOutputRightRight === SensorStatus.PaperPresent;
+  const someBackSensorsDetectPaper =
+    someBackSensorsUnderRollersDetectPaper ||
+    someBackSensorsPastRollersDetectPaper;
 
   if (scannerStatus.isScannerCoverOpen) {
     return { type: 'SCANNER_DISCONNECTED' };
   }
 
   if (scannerStatus.isPaperJam || scannerStatus.isJamPaperHeldBack) {
-    if (!frontHasPaper && !backHasPaper) {
+    if (!someFrontSensorsDetectPaper && !someBackSensorsDetectPaper) {
```

Comment:
This is stricter than before - intentional?

Reply:
It was intentional - I figured we'd want users to completely clear the scanner before considering the jam cleared. Though on second thought, we might not want to block on the back sensors past the rollers being clear, as per my comments elsewhere in this PR.

I think I'll switch to `!someFrontSensorsDetectPaper && !someBackSensorsUnderRollersDetectPaper`. Interestingly, with that change in place, we'll never be using the signal from the back sensors past the rollers.

Reply:
🤔 does make you wonder what those sensors are good for

Reply:
Updated in https://github.com/votingworks/vxsuite/pull/4395/commits/f2dc1adb0e2eafd59e96bc838a85fe2ee053e6ae

And yeah I've been wondering the same 🤔